daemon/k8s_watcher.go

```diff
@@ -443,46 +441,6 @@ func (d *Daemon) EnableK8sWatcher(reSyncPeriod time.Duration) error {
 
 	si.Start(wait.NeverStop)
 
-	_, nodesController := cache.NewInformer(
```

Comment:
What happen here? Why are we no longer watching for node events? How are we going to retrieve the podCIDR?

Reply:
The PodCIDR is retrieved independent of the node watcher, here:
https://github.com/cilium/cilium/blob/master/pkg/k8s/init.go#L112